src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
```

Comment:
The return type `Color` is not defined in the current context. Make sure that you have an enum `Color` defined with the possible color constants, or import it if it is defined in another package.